examples/smac/recurrent/centralised/run_qmix.py

```diff
@@ -70,31 +72,38 @@ def main(_: Any) -> None:
         time_delta=log_every,
     )
 
-    # distributed program
-    program = vdn.VDN(
+    # Distributed program
+    program = value_decomposition.ValueDecomposition(
         environment_factory=environment_factory,
         network_factory=network_factory,
+        mixer="qmix",
         logger_factory=logger_factory,
         num_executors=1,
-        exploration_scheduler_fn=LinearExplorationTimestepScheduler(
-            epsilon_start=1.0, epsilon_min=0.05, epsilon_decay_steps=50000
+        exploration_scheduler_fn=LinearExplorationScheduler(
+            epsilon_start=1.0, epsilon_min=0.05, epsilon_decay=4e-6
         ),
         optimizer=snt.optimizers.RMSProp(
             learning_rate=0.0005, epsilon=0.00001, decay=0.99
         ),
         checkpoint_subpath=checkpoint_dir,
-        batch_size=512,
-        executor_variable_update_period=100,
+        batch_size=32,
+        executor_variable_update_period=200,
         target_update_period=200,
-        max_gradient_norm=10.0,
-        eval_loop_fn=MonitorParallelEnvironmentLoop,
-        eval_loop_fn_kwargs={"path": checkpoint_dir, "record_every": 100},
+        max_gradient_norm=20.0,
+        min_replay_size=32,
+        max_replay_size=5000,
+        samples_per_insert=4,
+        sequence_length=20,
+        period=10,
+        evaluator_interval={"executor_episodes": 2},
```

Comment:
Can we make this closer the values in pymarl, I think it was 2000?

Reply:
Do you mean that the evaluator interval was 2000? I am happy to update that. In PyMARL sequence length is between 60 and 200 depending on the Map. I found that 200 used a  lot of RAM (45GB plus), which I think is impractical for normal users who want to run the algorithm locally. In my experiments I found that 20 used less than 16 GB of RAM and still trained fairly well. I think this is a more reasonable config for the example script. I don't want users to run the example and then have it crash because they ran out of RAM.